src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -905,10 +905,7 @@ <h5 class="col-sm-2">
 
 
     
-        <div id="statusMessage" class="alert alert-warning">
-            The feedback session has been updated.
-        </div>
-        <script type="text/javascript" src="/js/statusMessage.js"></script>
+        <div id="statusMessage" style="display: none;"></div>
```

Comment:
Ah, we should still show a status message if the feedback session was successfully modified. Your javascript still focuses the user to the status message (a blank message, in some cases) after editing.

Why is there a need to refresh the page in the test?